dist/runCall.js

```diff
@@ -89,19 +89,30 @@ module.exports = function (opts, cb) {
         case GETACCOUNT:
           {
             console.log('GETACCOUNT');
-            computeGetAccount(query);
+            computeGetAccount(query, function (fromAccount) {
+              var account = createAccount(fromAccount);
```

Comment:
Perhaps you could change `account` into `message`  as you did in `computeGetCode` as a convention for new protobuf messages?